### DISCUSSION TITLE
Hotkey "F5" to refresh, conflict in MC to "copy"

### DISCUSSION THREAD

**Discussion by RarogVA** [elementary/terminal#628] [Q&A] [answered]
Please delete the F5 hotkey to refresh window. I used Midnight Commander and often use F5 to copy files. That conflict are fire my ass))))
And also add option to rename tabs! All new tabs named by user and dont renemed permanently...

**Comment by jeremypw** [comment, marked as answer]
@RarogVA Thanks for the question!  The first part was reported in #617 for which a fix has been proposed in #623 and is awaiting review.  Hopefully it will be released before too long. You should open a separate feature request at https://github.com/elementary/terminal/issues for the second part (provided you cannot find a pre-existing request).

**Reply by RarogVA** [reply]
Done
